tests/CodeGen-MkBundle/Xamarin.Android.MakeBundle-UnitTests/Config.cs.in

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.IO;
+
+namespace Xamarin.Android.MakeBundle.UnitTests
+{
+	public static class Config
+	{
+		public static readonly string LlvmReadobj = Path.Combine ("@ANDROID_NDK_DIRECTORY@", "toolchains", "llvm", "prebuilt", "@HOST_ARCH@", "bin", "llvm-readobj@EXECUTABLE_EXTENSION@");
+		public static readonly string GccReadelf = Path.Combine ("@ANDROID_NDK_DIRECTORY@", "toolchains", "@OLD_TOOLCHAIN@", "prebuilt", "@HOST_ARCH@", "bin", "x86_64-linux-android-readelf@EXECUTABLE_EXTENSION@");
```

Comment:
As per 344022757b07d5bf1ba82c05b1171926b098266b, this should use `@"@ANDROID_NDK_DIRECTORY@"` (using @-strings), so that this works properly on Windows.